src/Settings.php

```diff
@@ -193,38 +209,44 @@ public function delete_option( $key = '' ) {
 		}
 
 		/**
-		 * Here is an example of removing settings from Events > Settings > General tab > "Map Settings" section
-		 * that are specific to Google Maps.
-		 */
-		public function remove_settings() {
-			// "Enable Google Maps" checkbox
-			$this->settings_helper->remove_field( 'embedGoogleMaps', 'general' );
-			// "Map view search distance limit" (default of 25)
-			$this->settings_helper->remove_field( 'geoloc_default_geofence', 'general' );
-			// "Google Maps default zoom level" (0-21, default of 10)
-			$this->settings_helper->remove_field( 'embedGoogleMapsZoom', 'general' );
-		}
-
-		/**
-		 * Adds a new section of fields to Events > Settings > General tab, appearing after the "Map Settings" section
-		 * and before the "Miscellaneous Settings" section.
+		 * Adds a new section of fields to Events > Settings > General tab, appearing before the "Miscellaneous Settings" section.
 		 *
-		 * TODO: Move it to where you want and update this docblock. If you like it here, just delete this TODO.
+		 * @since 1.0.0
 		 */
 		public function add_settings() {
-			$fields = [
-				// TODO: Settings heading start. Remove this element if not needed. Also remove the corresponding `get_example_intro_text()` method below.
-				'Example'   => [
-					'type' => 'html',
-					'html' => $this->get_example_intro_text(),
+			global $wp_roles;
```

Comment:
Why roles instead of a capability, such as a custom cap or a generic one like `manage_options`?
See https://wordpress.org/support/article/roles-and-capabilities/#capability-vs-role-table

Reply:
Because most users think in roles - and I didn't want a huge dropdown of capabilities in the settings?

Reply:
@Camwyn 

Why would Subscribers and other non-admins use this tool?
![image](https://user-images.githubusercontent.com/1812179/80638063-80f22f80-8a25-11ea-99a9-e90966955ce1.png)


There should be a filterable capability that Admins and above typically have and use `current_user_can()`

Feel free to ask around for others' opinions.